CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+### 1.100 - 2022-12-01
```

Comment:
```suggestion
### 1.100 - 2022-12-01

### engine

```

Since this change affects the `engine` package, the changes should be added under this header.